src_kt/java/arcs/util/BitVector.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.util
+
+/**
+ * Implementation of a bit vector as an array of bytes. Supports iterating over arbitrarily-sized
+ * windows (of length between 1 and 32).
+ */
+class BitVector(
```

Comment:
Would https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.native/-bit-set/index.html be helpful here?

Reply:
That's kotlin-native only, which means it won't work for JavaPEC/Particle Accelerator. We should stick to "common" kotlin stdlib that is present in both JVM and Native at least.